### DISCUSSION TITLE
Thermal Properties Database

### DISCUSSION THREAD

**Discussion by PhilipHouston1** [firemodels/cfast#2105] [General]
Where could I find the thermal properties database and how would I reference using these thermal properties for a research project?

**Comment by mcgratta** [comment]
There is no database. I would consult a heat transfer textbook or something like SFPE Handbook for thermal properties of common materials.

**Comment by drjfloyd** [comment]
materials.fsri.org has some data for common combustible construction materials
janaf.nist.gov (pure substances,specific heat, phase change)
webbook.nist.gov (pure substances,specific heat, phase change)
www.matweb.com (k, c, rho and other data for many materials)
your favorite search engine
